src/js/tabs/failsafe.js

```diff
@@ -317,9 +355,13 @@ TABS.failsafe.initialize = function (callback, scrollPosition) {
             }
 
             function save_feature_config() {
-                MSP.send_message(MSPCodes.MSP_SET_FEATURE_CONFIG, mspHelper.crunch(MSPCodes.MSP_SET_FEATURE_CONFIG), false, save_to_eeprom);
+                MSP.send_message(MSPCodes.MSP_SET_FEATURE_CONFIG, mspHelper.crunch(MSPCodes.MSP_SET_FEATURE_CONFIG), false, save_gps_rescue);
```

Comment:
Same here, we need to version check, and skip `save_gps_rescue()` if < 1.41.

Reply:
Hmm, this does not seem to be doing the version check.

Reply:
True, it seems something is wrong. I will recheck the code when I get home.